gui/wxpython/gui_core/forms.py

```diff
@@ -2015,7 +2015,7 @@ def __init__(self, parent, giface, task, id=wx.ID_ANY, frame=None, *args, **kwar
                     # check wildcard
                     try:
                         fExt = os.path.splitext(p.get("key_desc", ["*.*"])[0])[1]
-                    except:
+                    except Exception:
```

Comment:
IndexError?

Reply:
There can be `IndexError`, `OSError` and `AttributeError` all of which are applicable, even `TypeError` hence I thought I'd rather just put a catchall exception

Reply:
I think realistically p["key_desc"] can be empty, resulting in IndexError. I don't see how you can practically get the other ones given how p is created, and if it happens, it may be a programmer error and it's probably alright to let it slip through (and fix it).